test/test_matmul_cuda.py

```diff
@@ -526,6 +523,32 @@ def test_float8_scale_fast_accum(self, device) -> None:
         out_fp8_s, amax_fp8_s = torch._scaled_mm(x, y, scale_a=scale_a, scale_b=scale_b, use_fast_accum=True)
         self.assertEqual(out_fp8, out_fp8_s)
 
+    @unittest.skipIf(not scaled_mm_supported_device(), f8_msg)
+    @skipIfRocm()
+    @parametrize("use_fast_accum", [True, False])
+    def test_float8_rowwise_scaling_sanity(self, device, use_fast_accum: bool) -> None:
```

Comment:
would be nice to also test approximate numerics on more realistic data, but reasonable if that lives in float8_experimental

Reply:
added a test